tests/containers/skopeo.pm

```diff
@@ -0,0 +1,119 @@
+# SUSE's openQA tests
+#
+# Copyright 2023-2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: skopeo
+# Summary: Test basic skopeo commands.
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';    # used in select_serial_terminal
+use utils 'zypper_call';    # used in zypper_call
+use version_utils 'package_version_cmp';
+use version_utils 'is_transactional';
+use transactional;
+
+sub run {
+    my ($self, $args) = @_;
+
+    # Ensure skopeo is installed
+    # people make a $packages array which is populated with the packages that
+    # they want installed.
+
+
+    ######### Set required variables
+
+    # Required packages
+    my $packages = 'skopeo jq';    # and add whatever else should be missing with push function
+
+    select_serial_terminal();    # Select most suitable text console
+
+    # Set a variable for my remote image
+    my $remote_image = 'docker://registry.suse.com/bci/bci-busybox:latest';
+    # Set a variable for my working directory
+    my $workdir = '/tmp/test';
+
+    ######### Ensure pre-requisites
+
+    # Install required packages
+    # Make logic to account for SLE-Micro environment with transactional-update
+    is_transactional ? trup_install($packages) : zypper_call("--quiet in $packages", timeout => 120);
```

Comment:
You can `use containers::common qw(install_packages);` above and call it like this instead:

```suggestion
    install_packages(@packages);
```

After fixing remove any unneeded imports.

Reply:
Also, don't use `--quiet`, the `zypper_call` wrapper already contains those automatic flags.

Reply:
Done! Much handier call, thanks!